test/failure_handler/src/share/conf/windows.properties

```diff
@@ -57,7 +57,7 @@ native.stack.args=-c "~*kP n;qd" -p %p
 native.stack.params.repeat=6
 
 native.core.app=cdb
-native.core.args=-c ".dump /f core.%p;qd" -p %p
+native.core.args=-c ".dump /ma core.%p;qd" -p %p
```

Comment:
Just to double check that this is the right option. `/ma` terminates if there is "inaccessable memory" where `/mA` seems to be continue.  The `/f` option says all "accessible" memory which I assume it means skips/ignored inaccessible, is that right?

Reply:
`/ma` was the option suggested by `.dump`; looking at the documentation, `/mA` indeed seems to be a better choice. I'll update.

`/f` used to create a [full user-mode dump](https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/user-mode-dump-files#full-user-mode-dumps). I couldn't find any information about the behavior on inaccessible memory.
None of the options specify dumping inaccessible memory; I assume the failures only happen if some memory becomes inaccessible while producing the dump.

Reply:
Oh, I overlooked this. I think that is right, the old /f minidump generation worked around inaccessible memory. They were just "holes".